charts/rd-professional-api/values.preview.template.yaml

```diff
@@ -15,5 +15,6 @@ java:
     rd:
       secrets:
         - s2s-secret
+        - AppInsightsInstrumentationKey
```

Comment:
Will this enable all PR level app insights in higher environment ? if it is then AAT app insights will show PR endpoints info.

Reply:
yes it is for testing. I mnetioned in Abhijit Diwan:house_with_garden:  4:59 PM
https://github.com/hmcts/rd-professional-api/pull/704 have review on this ( note values-preview.yaml change temporary for testing)